src/cluster_legacy.h

```diff
@@ -344,6 +344,29 @@ static_assert(offsetof(clusterMsgLight, data) == 16, "unexpected field offset");
 
 #define CLUSTERMSG_LIGHT_MIN_LEN (sizeof(clusterMsgLight) - sizeof(union clusterMsgData))
 
+typedef struct {
+    char sig[4];       /* Signature "RCmb" (Cluster message bus). */
+    uint32_t totlen;   /* Total length of this message */
+    uint16_t ver;      /* Protocol version, currently set to CLUSTER_PROTO_VER. */
+    uint16_t notused1; /* full: port, light: notused1 */
+    uint16_t type;     /* Message type */
+    uint16_t notused2; /* full: count, light: notused2 */
+} clusterMsgUnknown;
+
+static_assert(offsetof(clusterMsgUnknown, sig) == offsetof(clusterMsg, sig), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, totlen) == offsetof(clusterMsg, totlen), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, ver) == offsetof(clusterMsg, ver), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, type) == offsetof(clusterMsg, type), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, notused1) == offsetof(clusterMsg, port), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, notused2) == offsetof(clusterMsg, count), "unexpected field offset");
+
+static_assert(offsetof(clusterMsgUnknown, sig) == offsetof(clusterMsgLight, sig), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, totlen) == offsetof(clusterMsgLight, totlen), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, ver) == offsetof(clusterMsgLight, ver), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, type) == offsetof(clusterMsgLight, type), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, notused1) == offsetof(clusterMsgLight, notused1), "unexpected field offset");
+static_assert(offsetof(clusterMsgUnknown, notused2) == offsetof(clusterMsgLight, notused2), "unexpected field offset");
```

Comment:
```suggestion
```
We already assert all the fields from msg = msgLight, so just asserting msg is equal to the main packet is sufficient.